### PR TITLE
Added wiki guide page in Resources => Guides => Wi-Fi Guide

### PR DESCRIPTION
**Pull request by abdularif0705**
### Did you read and follow article requirements
yes

### What are you trying to accomplish?
add a wifi guide pg

### How are you accomplishing it?
using the links in the issue for uwin wifi

### Is there anything reviewers should know?
I'm going to cite the websites that I pulled the info from. Gotta add how to connect to uwindsor

- [x] Is it safe to rollback this change if anything goes wrong?
yes

### CODE REVIEW

File: resources/guides/wifi_guide.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+---
+id: wifi_guide
+# title: Wi-Fi Guide # ![img](../../static/img/wifi-logo-blue-png-photo-resized.png 'Wifi Logo')
+sidebar_label: Wi-Fi Guide # ![img](../../static/img/wifi-logo-blue-png-photo-resized.png 'Wifi Logo')
+slug: /guides/wifi
+---
+# Wi-Fi Guide ![img](../../static/img/wifi-logo-blue-png-photo-resized.png 'Wifi Logo')
```
Comment (prairir) [inline resources/guides/wifi_guide.md]: This should be removed

File: resources/guides/wifi_guide.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
+---
+id: wifi_guide
+# title: Wi-Fi Guide # ![img](../../static/img/wifi-logo-blue-png-photo-resized.png 'Wifi Logo')
```
Comment (prairir) [inline resources/guides/wifi_guide.md]: This should be uncommented and not include the image tag

**Review by kilbouri — CHANGES_REQUESTED** [review]
resources/guides/wifi_guide.md
   Looks like a copy/paste, probably a bunch of information that doesn't apply to students that can be trimmed out.
   Also, there's a bit at the bottom about "Section breaks" that should be removed.

File: resources/guides/wifi_guide.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+If you are a confirmed student and never used my.uwindsor.ca before, you can activate your UWin Account on this site: my.uwindsor.ca/web/uw/activate-account
+
+### The IT Service Desk
+
+The IT Service Desk is located in the University Computer Centre (south entrance of the CAW building). ITS can help with any UWinID access or password issues, student email issues, and any information as to processes and policies involving University computers and networks on campus. ITS can also be reached at 519-253-3000 x4440 and is open 8:30AM to 4:30PM. Students also have the ability to report an issue or question at www.uwindsor.ca/its anytime.
+
+
+
+## Section Breaks
```
Comment (kilbouri) [inline resources/guides/wifi_guide.md]: Looks like is an accidental artifact of editing a template. Should be removed.

File: resources/guides/wifi_guide.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+- **uwinsecure**
+- **uwindsor**
+
+Which should you use? The short answer is “Connect to eduroam”. The eduroam wireless network is available to all wireless users (Students/Faculty/Staff/guests). Plus it has the additional benefit that users from other eduroam institutions (many universities, for example) can also connect to the University of Windsor’s wireless network using their home institution’s crededentials.
+
+## Eduroam - The Preferred Wireless Network
+![alt text](https://www.uwindsor.ca/itservices/sites/uwindsor.ca.itservices/files/246studentlaw_sm-retouched_crop.jpg "girl connecting to the right choice")
+
+**Eduroam** is a network used by many universities, and your UWindsor sign in will work at any campus that uses eduroam.
+1. Open Wi-Fi settings on your device and select the “eduroam” network
```
Comment (prairir) [inline resources/guides/wifi_guide.md]: I think we should split it up by device or OS. for example, the instructions for linux are much different than something like windows

File: resources/guides/wifi_guide.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+You use your UWindsor ID to sign into your UWinsite Student, Blackboard, MySuccess, etc. Your UWindsor email address is your ID @uwindsor.ca and sign-in is via Gmail.
+
+To activate your ID, you will require your 9-digit Student Number and 4-digit Personal Access Code (PAC). Find these by searching your inbox, or on your acceptance letter. Visit the Leddy Library or the Computer Centre to use an internet-enabled computer and activate your account.
+
+If you are a confirmed student and never used my.uwindsor.ca before, you can activate your UWin Account on this site: my.uwindsor.ca/web/uw/activate-account
+
+### The IT Service Desk
+
+The IT Service Desk is located in the University Computer Centre (south entrance of the CAW building). ITS can help with any UWinID access or password issues, student email issues, and any information as to processes and policies involving University computers and networks on campus. ITS can also be reached at 519-253-3000 x4440 and is open 8:30AM to 4:30PM. Students also have the ability to report an issue or question at www.uwindsor.ca/its anytime.
```
Comment (kilbouri) [inline resources/guides/wifi_guide.md]: This seems to be false. Fact checking needed ;)

File: resources/guides/wifi_guide.md (hunk 3 above)
Comment (prairir) [inline resources/guides/wifi_guide.md]: this section and the `Further Reading` section arent needed

**Review by prairir — CHANGES_REQUESTED** [review]

File: resources/guides/wifi_guide.md (hunk 4 above)
Comment (kilbouri) [reply resources/guides/wifi_guide.md]: +1, bump.

Comment (abdularif0705) [reply resources/guides/wifi_guide.md]: ok I only know what to do for Windows so maybe someone else can add on for Linux

Comment (prairir) [reply resources/guides/wifi_guide.md]: Can we add a section for Iphone or android? Please add issues for platforms that you dont have. for example, an issue like `Wifi instructions for linux`

Comment (abdularif0705) [reply resources/guides/wifi_guide.md:22]: ok sounds good

**Review by abdularif0705 — COMMENTED** [review]
It's showing unnecessary files that have 0 changes lmao

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 6]
```diff
@@ -1,3 +1,20 @@
+<<<<<<< HEAD
```
Comment (kilbouri) [inline .github/ISSUE_TEMPLATE/bug-report.md]: You need to remove these. They were added by a merge conflict and should be removed. Same goes for the `=======` and `>>>>>>> master` below.

File: .github/ISSUE_TEMPLATE/code-maintenance.md [hunk 7]
```diff
@@ -1,3 +1,19 @@
+<<<<<<< HEAD
```
Comment (kilbouri) [inline .github/ISSUE_TEMPLATE/code-maintenance.md]: Remove these (see comment in bug-report.md)

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 8]
```diff
@@ -1,3 +1,19 @@
+<<<<<<< HEAD
```
Comment (kilbouri) [inline .github/ISSUE_TEMPLATE/feature-request.md]: Remove these (see comment in bug-report.md)

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 9]
```diff
@@ -1,3 +1,18 @@
+<<<<<<< HEAD
```
Comment (kilbouri) [inline .github/PULL_REQUEST_TEMPLATE.md]: Remove these (see comment in bug-report.md)

File: ARCHITECTURE.md [hunk 10]
```diff
@@ -1,3 +1,119 @@
+<<<<<<< HEAD
```
Comment (kilbouri) [inline ARCHITECTURE.md]: Remove these (see comment in bug-report.md)

File: academics/ImportantDates.md [hunk 11]
```diff
@@ -1,3 +1,38 @@
+<<<<<<< HEAD
```
Comment (kilbouri) [inline academics/ImportantDates.md]: Remove these (see comment in bug-report.md)

File: README.md [hunk 12]
```diff
@@ -1,3 +1,38 @@
+<<<<<<< HEAD
```
Comment (kilbouri) [inline README.md]: Remove these (see comment in bug-report.md)

File: academics/academics_sidebars.js [hunk 13]
```diff
@@ -1,3 +1,22 @@
+<<<<<<< HEAD
```
Comment (kilbouri) [inline academics/academics_sidebars.js]: Remove these (see comment in bug-report.md)

File: academics/awards/awardApplication.md [hunk 14]
```diff
@@ -1,3 +1,19 @@
+<<<<<<< HEAD
```
Comment (kilbouri) [inline academics/awards/awardApplication.md]: Remove these (see comment in bug-report.md)

File: babel.config.js [hunk 15]
```diff
@@ -1,3 +1,9 @@
+<<<<<<< HEAD
```
Comment (kilbouri) [inline babel.config.js]: Remove these (see comment in bug-report.md)

File: careers/careers_sidebars.js [hunk 16]
```diff
@@ -1,3 +1,41 @@
+<<<<<<< HEAD
```
Comment (kilbouri) [inline careers/careers_sidebars.js]: Remove these (see comment in bug-report.md)

File: courses/OfferedCourses.md [hunk 17]
```diff
@@ -1,3 +1,76 @@
+<<<<<<< HEAD
```
Comment (kilbouri) [inline courses/OfferedCourses.md]: Remove these (see comment in bug-report.md)

File: courses/courses_sidebars.js [hunk 18]
```diff
@@ -1,3 +1,13 @@
+<<<<<<< HEAD
```
Comment (kilbouri) [inline courses/courses_sidebars.js]: Remove these (see comment in bug-report.md)

File: css/constitution.md [hunk 19]
```diff
@@ -1,3 +1,629 @@
+<<<<<<< HEAD
```
Comment (kilbouri) [inline css/constitution.md]: Remove these (see comment in bug-report.md)

File: css/css_sidebars.js [hunk 20]
```diff
@@ -1,3 +1,13 @@
+<<<<<<< HEAD
```
Comment (kilbouri) [inline css/css_sidebars.js]: Remove these (see comment in bug-report.md)

File: docusaurus.config.js [hunk 21]
```diff
@@ -1,3 +1,224 @@
+<<<<<<< HEAD
```
Comment (kilbouri) [inline docusaurus.config.js]: Remove these (see comment in bug-report.md)

File: newsletter/2021-03-17-welcome.md [hunk 22]
```diff
@@ -1,3 +1,21 @@
+<<<<<<< HEAD
```
Comment (kilbouri) [inline newsletter/2021-03-17-welcome.md]: Remove these (see comment in bug-report.md)

File: package.json [hunk 23]
```diff
@@ -1,3 +1,40 @@
+<<<<<<< HEAD
```
Comment (kilbouri) [inline package.json]: Remove these (see comment in bug-report.md)

File: resources/guides/style_guide.md [hunk 24]
```diff
@@ -1,3 +1,440 @@
+<<<<<<< HEAD
```
Comment (kilbouri) [inline resources/guides/style_guide.md]: Remove these (see comment in bug-report.md)

File: resources/resources_sidebars.js [hunk 25]
```diff
@@ -1,3 +1,18 @@
+<<<<<<< HEAD
```
Comment (kilbouri) [inline resources/resources_sidebars.js]: Remove these (see comment in bug-report.md)

File: sidebars.js [hunk 26]
```diff
@@ -1 +1,6 @@
+<<<<<<< HEAD
```
Comment (kilbouri) [inline sidebars.js]: Remove these (see comment in bug-report.md)

File: src/css/custom.css [hunk 27]
```diff
@@ -1,3 +1,112 @@
+<<<<<<< HEAD
```
Comment (kilbouri) [inline src/css/custom.css]: Remove these (see comment in bug-report.md)

File: src/pages/index.js [hunk 28]
```diff
@@ -1,3 +1,112 @@
+<<<<<<< HEAD
```
Comment (kilbouri) [inline src/pages/index.js]: Remove these (see comment in bug-report.md)

File: src/pages/styles.module.css [hunk 29]
```diff
@@ -1,3 +1,42 @@
+<<<<<<< HEAD
```
Comment (kilbouri) [inline src/pages/styles.module.css]: Remove these (see comment in bug-report.md)

File: yarn.lock [hunk 30]
```diff
@@ -1,3 +1,10375 @@
+<<<<<<< HEAD
```
Comment (kilbouri) [inline yarn.lock]: Remove these (see comment in bug-report.md)

**Review by kilbouri — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 6 above)
Comment (abdularif0705) [reply .github/ISSUE_TEMPLATE/bug-report.md]: Thanks g

File: resources/guides/wifi_guide.md [hunk 31]
```diff
@@ -0,0 +1,57 @@
...
+
+### For Linux
+
+The **uwindsor** wireless network is available for use but should only be used as a **last resort**. Unlike **eduroam** and **uwinsecure**, the **uwindsor** network does not encrypt the network traffic. So, for example, if you are accessing network applications using a protocol which is insecure, (eg. http instead of https), then everyone else using the **uwindsor** network in the vicinity could view your network activity. The wireless network called **uwindsor** is not secure.
+
+### [For Residence Students](https://www.uwindsor.ca/residence/wifi)
+
+While Eduroam is the primary campus-wide Wi-Fi network, you may want to use UWinResLife in your hall for heavier use (i.e. streaming or downloading). Sign in using your UWindsor ID (just the first part - _not @uwindsor.ca_), and your UWinID password.
+
+### Activating Your UWindsor Account
```
Comment (prairir) [inline resources/guides/wifi_guide.md]:
```suggestion
## Activating Your UWindsor Account
```

File: resources/guides/wifi_guide.md [hunk 32]
```diff
@@ -0,0 +1,57 @@
...
+
+If prompted to trust a certificate after signing in (mainly iPhone) click trust.
+
+### For Macs
+
+### For Linux
+
+The **uwindsor** wireless network is available for use but should only be used as a **last resort**. Unlike **eduroam** and **uwinsecure**, the **uwindsor** network does not encrypt the network traffic. So, for example, if you are accessing network applications using a protocol which is insecure, (eg. http instead of https), then everyone else using the **uwindsor** network in the vicinity could view your network activity. The wireless network called **uwindsor** is not secure.
+
+### [For Residence Students](https://www.uwindsor.ca/residence/wifi)
```
Comment (prairir) [inline resources/guides/wifi_guide.md]:
```suggestion
## [For Residence Students](https://www.uwindsor.ca/residence/wifi)
```

**Review by prairir — CHANGES_REQUESTED** [review]
Overall this is great work. I just want those headers to be a bit larger

File: academics/awards/awardApplication.md [hunk 33]
```diff
@@ -10,3 +10,5 @@ Log into your [UWinsite Student][uwinsite] and complete the Award Profile. The U
 [awardsearch]: https://www.uwindsor.ca/studentawards/awards-search?ACAD_CAREER=UGRD&UW_FAC=SCIENCE
 [uwinsite]: https://student.uwindsor.ca
 [guide]: https://www.uwindsor.ca/studentawards/sites/uwindsor.ca.studentawards/files/uwinaward_application_profile_instructions_for_students_3.pdf
+
+testing
```
Comment (kilbouri) [inline academics/awards/awardApplication.md]: Reason? Probably should be removed

File: courses/OfferedCourses.md [hunk 34]
```diff
@@ -53,7 +53,7 @@ hide_table_of_contents: true
 | COMP-4150   |     Advanced and Practical Database Systems      | Offered     | Not Offered | Not Offered | 💛           |                       COMP-3150, COMP-3300                        |
 | COMP-4200   |          Mobile Application Development          | Not Offered | Offered     | Not Offered | 💛           |                       COMP-3150, COMP-3220                        |
 | COMP-4220   |            Agile Software Development            | Offered     | Not Offered | Not Offered | 💛           |                             COMP-3220                             |
-| COMP-4250   |      Big Data Analytics and Database Design      | Not Offered | Offered     | Not Offered | 💛           |                             COMP-3150                             |
+| COMP-4250   |      Big Data Analytics and Database Design      | Not Offered | Offered     | Not Offered | 💛💜         |                             COMP-3150                             |
```
Comment (kilbouri) [inline courses/OfferedCourses.md:56]: Source? Just want to be sure this is correct and not a typo.

File: css/constitution.md [hunk 35]
```diff
@@ -30,6 +30,332 @@ Society Membership shall consist of:
...
+1. All students who are currently enrolled on a full time basis at the
+   University of Windsor majoring in a program offered by the department of
+   Computer Science.
+2. Any student at the University of Windsor, who is not otherwise
+   covered by part a, may purchase a membership for **$20 per semester**.
+
+### Article 2.
+
+#### Section 1 - Positions
+
```
Comment (kilbouri) [inline css/constitution.md]: Looks like everything above this is a duplicate of everything below. Possible residue from a merge conflict.

**Review by kilbouri — CHANGES_REQUESTED** [review]
I do question why all the dependency changes in `yarn.lock`. Its possible something updated, just want to know why there were so many dependencies were changed.

File: courses/OfferedCourses.md (hunk 34 above)
Comment (abdularif0705) [reply courses/OfferedCourses.md:56]: I talked to my advisor about required courses and she told me that I need Big Data as it is required. And I'm part of the Business & CS program

**Review by prairir — APPROVED** [review]

**Comment by prairir** [comment]
LGTM :rocket:

File: css/constitution.md (hunk 35 above)
Comment (abdularif0705) [reply css/constitution.md]: nope, this is the exact file I copied from the master branch

**Comment by abdularif0705** [comment]
> I do question why all the dependency changes in `yarn.lock`. Its possible something updated, just want to know why there were so many dependencies were changed.

no idea, all I ran was `yarn install --dev` and `yarn prettier --write .`

**Review by kilbouri — APPROVED** [review]

**Review by kilbouri — COMMENTED** [review]
I'm confused what changed in the Constitution. The merge conflict is very confusing, an entire article seems to have been deleted.

**Comment by jere-mie** [comment]
> I'm confused what changed in the Constitution. The merge conflict is very confusing, an entire article seems to have been deleted.

This is because we just updated the constitution to the newest version which was approved at the last CSS meeting. Whatever is currently in master should be used.

**Pull request merged**